vcell-core/src/main/java/org/vcell/sedml/SEDMLExporter.java

```diff
@@ -512,6 +538,7 @@ private void translateBioModelToSedML(String savePath, String sBaseFileName, boo
 
 							for (String unscannedParamName : unscannedParamHash.values()) {
 								SymbolTableEntry ste = getSymbolTableEntryForModelEntity(mathSymbolMapping, unscannedParamName);
+								assert ste != null;
```

Comment:
throw new RuntimeException("failed to retrieve vcell parameter from unscanned SEDML parameter "+unscannedParamName+" using mathSymbolMapping);

Reply:
we should probably avoid assert statements in production code.

Reply:
I would clean those up once we also do proper log4j logging in SEDMLexporter... class also needs some refactoring of giant method that does the actual export